### ISSUE TITLE
Problems with getInitialProps

### ISSUE THREAD

**Issue by tizzle** [isaachinman/next-i18next#13]
**Describe the bug**
When using a custom server together with next-i18next the props from `getInitialProps` are not passed to the component.

**To Reproduce**
Steps to reproduce the behaviour:
1. Clone or download this repository
2. In terminal do `cd example`
3. Install dependencies: `yarn`
4. I had to install next-i18next as well: `yarn add next-i18next`
5. Add a new page `content.js` in `/pages`
```
import React from 'react';
import { withNamespaces } from '../i18n'

const ContentContainer = ({
  slug
}) => {
  return (
    <div>
      Lorem ipsum
      {slug}
    </div>
  );
}

ContentContainer.getInitialProps = ({
  query: { slug }
}) => {
  console.log(slug);
  return {
    slug
  };
}

export default withNamespaces('common')(ContentContainer);
```
6. Add custom route handling to `server.js`:
```
...
  nextI18NextMiddleware(app, server)

  server.get('/content/:slug', (req, res) => {
    app.render(
      req,
      res,
      '/content',
      {
        slug: req.params.slug,
      },
    )
  })

  server.get('*', (req, res) => handle(req, res))
...
```
7. Start application: `yarn run dev`

**Expected behaviour**
When visiting the route `/content/testSlug` the page should display `Lorem Ipsum testSlug`, but just shows `Lorem Ipsum`. The `slug` property is undefined. When removing all `next-i18next` code from the example (_app.js, index.js, content.js) it works as expected.

**OS (please complete the following information):**
 - Device: MacBook Pro (15-inch, 2017)
 - Browser: Chrome Version 70.0.3538.110

**Comment by tizzle** [comment]
I did some digging in the code and i think i found the culprit.

In `app-with-tranlation.js` it's necessary to return `pageProps: { ...pageProps }` instead of '...pageProps'.

**Comment by tizzle** [comment]
There's already a PR for that: https://github.com/isaachinman/next-i18next/pull/12

**Comment by isaachinman** [comment]
@tizzle I've just merged #12, that indeed fixes the problem here. I also want to thank you for fantastic bug reporting.
